phylo2owl.py

```diff
@@ -117,7 +128,6 @@
     sys.stderr.write("\n")
 
 # Step 6. Write out each node on the tree.
-import inspect
```

Comment:
Oops, sorry about that! Moved into its own commit, @22b147a -- or should I remove this from this pull request entirely?